README.md

```diff
@@ -16,6 +16,7 @@ We follow a [Code of Conduct](docs/code-of-conduct.md).
 		* [By using DNSMASQ.](#by-using-dnsmaws)
 		* [Or by using the hostfile.](#or-by-using-the-hostfile)
 	* [3). Now, setup your projects!](#3-now-setup-your-projects)
+* [Update from exsisting environment.](#update)
```

Comment:
Fix typo: existing.
Also: `Update your existing installation` would be a better fit.